src/ui/public/agg_types/__tests__/param_types/_field.js

```diff
@@ -24,4 +27,38 @@ describe('Field', function () {
       expect(aggParam).to.be.a(BaseAggParam);
     });
   });
+
+  describe('getFieldOptions', function () {
+    it('should return only aggregatable fields', function () {
+      const aggParam = new FieldAggParam({
+        name: 'field'
+      });
+
+      const fields = aggParam.getFieldOptions({
+        getIndexPattern: () => indexPattern
+      });
+      for (let field of fields) {
```

Comment:
Add an assertion that that fields is not empty, otherwise an empty array would pass this test.